pylint/extensions/check_elif.py

```diff
@@ -38,37 +38,29 @@ def __init__(self, linter=None):
         self._init()
 
     def _init(self):
-        self._elifs = []
-        self._if_counter = 0
+        self._elifs = {}
 
     def process_tokens(self, tokens):
-        # Process tokens and look for 'if' or 'elif'
-        for _, token, _, _, _ in tokens:
+        """Process tokens and look for 'if' or 'elif'"""
+        for _, token, begin, _, _ in tokens:
             if token == "elif":
-                self._elifs.append(True)
+                self._elifs[begin] = True
             elif token == "if":
-                self._elifs.append(False)
+                self._elifs[begin] = False
```

Comment:
Do we need these to be `True` or `False` anywhere else than in the `visit_if`? For maintainability we might want to rename them to `"elif"` and `"if"` and just check for string equality.

Reply:
e1fe1672